regression_tests/src/venv_helper.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function create_and_activate_venv() {
```

Comment:
Inconsistent with the functions below. Probably just remove the "function" keyword as it's not necessary.